Makefile

```diff
@@ -39,17 +36,20 @@ coverage: ## Run pytest with coverage report.
 	coverage report -m
 	coverage html
 
+.PHONY: installdocs
+installdocs: ## Generate documentation.
```

Comment:
what's the diff between this and `make docs`? they both say `## Generate documentation.`

Reply:
It "s install the Python dependencies